system-tests/README.md

```diff
@@ -0,0 +1,42 @@
+## System tests
+
+System tests run a sample EDC connector with the Identity Hub extension and a DID server with sample DID documents for the connector and an external authority using docker. The test checks the correct functioning of the Identity Hub CLI together with the CredentialsVerifier.
```

Comment:
first sentence with multiple `and`/`with` clauses is hard to parse

Reply:
I reworded this section a bit. Please have a look.